client/src/components/categorical/category/index.js

```diff
@@ -7,27 +7,28 @@ import {
   Button,
   Tooltip,
   Icon,
-  Position
+  Position,
 } from "@blueprintjs/core";
 import CategoryFlipperLayout from "./categoryFlipperLayout";
 import AnnoMenu from "./annoMenuCategory";
 import AnnoDialogEditCategoryName from "./annoDialogEditCategoryName";
 import AnnoDialogAddLabel from "./annoDialogAddLabel";
 
-import * as globals from "../../globals";
-import maybeTruncateString from "../../util/maybeTruncateString";
+import * as globals from "../../../globals";
+import maybeTruncateString from "../../../util/maybeTruncateString";
 
-@connect(state => ({
+@connect((state) => ({
   colorAccessor: state.colors.colorAccessor,
```

Comment:
this has the same bug I just fixed in histograms - it will re-render every time the color accessor changes.

Suggest changing this to a bool: `isColorAccessor`, using the same method I used in brushableHistogram/index.js